projects/zabbix/playbooks/vagrant.retry

```diff
@@ -0,0 +1,9 @@
+apigate.vagrant.localdomain
```

Comment:
this a junk file! we should delete it, and maybe add a pattern for `vagrant.retry` to `.gitignore`.